.gitignore

```diff
@@ -1,5 +1,5 @@
 *.log
-/node_modules/
 /dist/
+/node_modules/
```

Comment:
did you intentionally move this or some IDE tool of yours? no big deal just wondering about the change

Reply:
Ah, this was me adding back in `/dist` after I removed it in my fork to get the small change I originally made to pass back the zxcvbn feedback. I shouldn't have been exposed in this PR. I'm reverting it back to the format you originally had it in.